### PR TITLE
Add guide: Enable forging - Closes #498

### LINKED ISSUE #498: New guide: Enable forging
**Describe the solution you'd like**
Add a guide to SDK docs that explains how to enable forging for a particular delegate on a node.

Should be very similar to the corresponding guide in the Core docs.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #498

### CODE REVIEW

File: modules/ROOT/pages/guides/forging.adoc [hunk 1]
```diff
@@ -0,0 +1,174 @@
+= Enable forging
+Mona Bärenfänger <mona@lightcurve.io>
+:toc:
+:url_commander_commands: reference/lisk-commander/user-guide/commands.adoc
+:url_elements_crytpography: reference/lisk-elements/packages/cryptography.adoc
+:url_best_practices: best-practices.adoc
+
+== What is forging?
+
+Forging in DPoS networks is the same as mining in PoW network: It is the process of adding new blocks to the chain.
```
Comment (chris529) [inline modules/ROOT/pages/guides/forging.adoc]:
```suggestion
Forging in DPoS networks is the same as mining in a PoW network: It is the process of adding new blocks to the chain.
```

File: modules/ROOT/pages/guides/forging.adoc [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+In DPoS system, each account that has enough balance to send a "register delegate" transaction can register a new delegate on the network.
```
Comment (chris529) [inline modules/ROOT/pages/guides/forging.adoc]:
```suggestion
In a DPoS system, each account that has an adequate enough balance to send a "register delegate" transaction can register a new delegate on the network.
```

File: modules/ROOT/pages/guides/forging.adoc [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+Other accounts can now vote for delegates by sending "cast votes" transactions.
+
+The 101 delegates with the most accumulated vote weight are in the active forging positions and each of them is allowed to forge one block in each round (a round lasts 101 blocks, then a new round starts).
```
Comment (chris529) [inline modules/ROOT/pages/guides/forging.adoc]:
```suggestion
The 101 delegates with the most accumulated vote weight are in the active forging positions, and each of them is allowed to forge one block in each round, (a round lasts for 101 blocks, after this then a new round starts).
```

File: modules/ROOT/pages/guides/forging.adoc [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+As reward for securing the network, the forging nodes receive the transaction fees and the block rewards of the forged blocks and their included transactions.
```
Comment (chris529) [inline modules/ROOT/pages/guides/forging.adoc]:
```suggestion
As a reward for securing the network, the forging nodes receive the transaction fees and the block rewards of the forged blocks and their included transactions.
```

File: modules/ROOT/pages/guides/forging.adoc [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+----
+
+<1> Contains options for the chain module.
+<2> Contains forging options for delegates.
+<3> Forces forging to be 'on'.
+This is only used on local development networks.
+<4> The list of delegates who are allowed to forge on this node.
+To successfully enable forging for a delegate, the public key and the encrypted passphrase need to be deposited here as a JSON object.
+<5> Contains options for the API module.
+<6> Replace with the IP address which will be used to access the node
```
Comment (chris529) [inline modules/ROOT/pages/guides/forging.adoc]:
```suggestion
<6> Replace with the IP address which will be used to access the node.
```

File: modules/ROOT/pages/guides/forging.adoc [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+
+TIP: Restart the node apply the changes in the config.
```
Comment (chris529) [inline modules/ROOT/pages/guides/forging.adoc]:
```suggestion
TIP: Restart the node and apply the changes in the config.
```

File: modules/ROOT/pages/guides/forging.adoc [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+          "publicKey": "YYYYYYYYY",
+          "password": "XXX",
+          "forging": false
+      }'
+----
+
+* `publicKey` is the key for the delegate which is required to be enabled/disabled.
+* `password` is the password used to encrypt your passphrase in the config.
+* `forging` is the boolean value to enable or disable the forging.
+* The HTTP Port can be different based on your configuration, therefore it is recommended to check the `httpPort` in your config/
```
Comment (chris529) [inline modules/ROOT/pages/guides/forging.adoc]:
```suggestion
* The HTTP Port can be different based on your configuration, therefore it is recommended to check the `httpPort` in your config.
```

**Review by chris529 — CHANGES_REQUESTED** [review]
Excellent, just a few minor issues ;-)

File: modules/ROOT/pages/guides/forging.adoc (hunk 6 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/forging.adoc]:
```suggestion
TIP: Restart the node to apply the changes in the config.
```

**Review by chris529 — APPROVED** [review]
Perfect.. :-)

**Pull request merged**